package.json

```diff
@@ -36,11 +36,11 @@
     "lint:md": "remark --frail --quiet .",
     "release": "pnpm --silent build && changeset publish",
     "skuba": "pnpm --silent build && pnpm --silent skuba:exec",
-    "skuba:exec": "node --env-file=.env lib/skuba",
+    "skuba:exec": "node --experimental-vm-modules lib/skuba",
```

Comment:
Using the .env file was breaking when using a VSCode debug terminal, because that sends NODE_OPTIONS. I can't see any downsides to inlining it?